monkey/infection_monkey/model/victim_host_generator_test.py

```diff
@@ -0,0 +1,44 @@
+from unittest import TestCase
+from infection_monkey.model.victim_host_generator import VictimHostGenerator
+from common.network.network_range import CidrRange, SingleIpRange
+
+
+class VictimHostGeneratorTester(TestCase):
+
+    def setUp(self):
+        self.test_ranges = [CidrRange("10.0.0.0/28", False),  # this gives us 15 hosts
```

Comment:
Unit tests should not have side effects (if multiple tests run in the same scope and alter it it's no longer called unit test it's "integration" tests). In theory you should've created a generator object for each test, but I don't insist on it, just have it in mind when writing future tests.